src/Components/Reports/Common/ExecutiveSecondPage.js

```diff
@@ -1,97 +1,59 @@
-import { AngleDoubleDownIconConfig } from '@patternfly/react-icons/dist/js/icons/angle-double-down-icon';
-import { AngleDoubleUpIconConfig } from '@patternfly/react-icons/dist/js/icons/angle-double-up-icon';
-import { AutomationIconConfig } from '@patternfly/react-icons/dist/js/icons/automation-icon';
-import { InProgressIconConfig } from '@patternfly/react-icons/dist/js/icons/in-progress-icon';
-import { CheckCircleIconConfig } from '@patternfly/react-icons/dist/js/icons/check-circle-icon';
-import styles from './styles';
-import { DescriptionList } from './DescriptionList';
+import { InstancesTable } from './InstancesTable';
+import propTypes from 'prop-types';
 
-export const ExecutiveSecondPage = () => {
+export const ExecutiveSecondPage = ({ data }) => {
 
-    const stateDescription = [
-        {
-            title: 'Optimized',
-            description: 'Performing at an optimal level',
-            iconPath: CheckCircleIconConfig.svgPath,
-            iconScale: 0.014,
-            fillColor: 'green'
-        },
-        {
-            title: 'Under pressure',
-            description: 'Peaking occasionally',
-            iconPath: TachometerAltIconConfig.svgPath,
-            iconScale: 0.014,
-            fillColor: '#030303'
-        },
-        {
-            title: 'Undersized',
-            description: 'Using more than 80% of system resources',
-            iconPath: AngleDoubleDownIconConfig.svgPath,
-            iconScale: 0.014,
-            fillColor: 'red'
-        },
-        {
-            title: 'Oversized',
-            description: 'Using less than 20% of system resources',
-            iconPath: AngleDoubleUpIconConfig.svgPath,
-            iconScale: 0.014,
-            fillColor: '#f09800'
-        },
-        {
-            title: 'Idling',
-            description: 'Consuming less than 5% of resources',
-            iconPath: AutomationIconConfig.svgPath,
-            iconScale: 0.008,
-            fillColor: '#030303'
-        },
-        {
-            title: 'Waiting for data',
-            description: 'Data has not been received or is being processed. Initial data processing takes up to 24 hours.',
-            iconPath: InProgressIconConfig.svgPath,
-            iconScale: 0.008,
-            fillColor: '#2B9AF3'
-        }
-    ];
+    const { current: currentData, suggested: suggestedData,
+        historical: historicalData } =  data?.instance_types_highlights;  /* eslint-disable-line camelcase */
+    const { stale_count: staleCount } = data?.meta;
 
-    const conditionsDescription = [
-        {
-            title: 'CPU pressure',
-            description: 'CPU registered peaks higher than 20% over several one-minute time periods'
+    const instanceDetails = {
+        current: {
+            heading: 'Current instance types',
+            description: 'This is your current state, representing the instances that are already reporting data.',
```

Comment:
Maybe something like, because we are representing the states of the systems?
~~~
This is the current state of systems and, represents the instances that are already reporting data.
~~~

Reply:
@upadhyeammit we had a discussion around the titles and description and [here is the doc](https://docs.google.com/document/d/1X75_NVGHKkwgS3ZVkXVe4oRzrUj402YRN18D14ZJoMo/edit#heading=h.qln18xg41vap) with the final details. I have followed the same in the code. But in case you have got suggestion can you please put on the doc I linked and tag pau for confirming the same?

Reply:
@upadhyeammit we had a discussion around the titles and description and [here is the doc](https://docs.google.com/document/d/1X75_NVGHKkwgS3ZVkXVe4oRzrUj402YRN18D14ZJoMo/edit#heading=h.qln18xg41vap) with the final details. I have followed the same in the code. But in case you have got suggestion can you please put on the doc I linked and tag pau for confirming the same?

Reply:
I can see there has been verbose discussion but I can't find this string anywhere there. I suggest to go with decided changes for now as its not blocker but maybe nice to have change.

Reply:
okay sure @upadhyeammit actually I updated the strings again today based on the file so above string is outdated. You will find the strings based on the document I shared.

Reply:
@upadhyeammit let me know if I can resolve this thread or if you have any further questions! 🙌🏻